.github/workflows/coding.yml

```
@@ -28,7 +28,7 @@
       #  run: npm run lint  # Assuming you have a lint script in your package.json  
   
   # TEST
-  test-scala-coding:  
+  test-scala:  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)